<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDairyEditWidget</class>
 <widget class="QWidget" name="CDairyEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>标题：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTitle"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>标签：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTag">
       <item>
        <property name="text">
         <string>无标签</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>日期：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDateTime">
       <property name="text">
        <string>2019年9月14日 09:33</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelWeekDay">
       <property name="text">
        <string> 周六</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>天气：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxWeather">
       <item>
        <property name="text">
         <string>晴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>多云</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>阴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>阵雨</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>小雨</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大雨</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>雷阵雨</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>沙尘</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>小雪</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大雪</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>冰雹</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CDairyEdit" name="dairyEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDairyEdit</class>
   <extends>QTextEdit</extends>
   <header>DairyEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
